src/universal/modules/userDashboard/components/UserActionList/ActionDragLayer.js

```diff
@@ -0,0 +1,56 @@
+import React, {Component, PropTypes} from 'react';
+import UserActionListItem from './UserActionListItem';
+import {DragLayer as dragLayer} from 'react-dnd';
+import ui from 'universal/styles/ui';
+
+const layerStyles = {
+  position: 'fixed',
+  pointerEvents: 'none',
+  zIndex: 500,
+  left: 0,
+  top: 0,
+  width: ui.dashActionsWidth,
+  minHeight: '15rem',
```

Comment:
@ackernaut i can't find any other way to make this pretty when we drag. can you think of something cleaner?